spec/index.html

```diff
@@ -1802,42 +1814,59 @@ <h3>RDF Triples Constructors</h3>
         |curPredicate| <var>N</var> .</span>
     </p>
 
-    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms:</span></h4>
+    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms</span></h4>
 
     <p>Beginning the <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> production
       records the |curSubject| and |curPredicate|.
```

Comment:
Small errata to RDF 1.1:

- It does not say that `curSubject` and `curPredicate` are stacks, or alternative the parser state is a stack.
- It says in 7.3 "The `curSubject` is bound to the [subject](https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/62.html#grammar-production-subject) production." True but it is more than just `subject` even in RDF 1.1 (subject position, `blankNodePropertyList`, property lists in the object position and collections). RDF 1.2 adds the `tripleTerm`, `reifiedTriple`, `annotation` and `annotationBlock` cases.

Reply:
Neither `curSubject` nor `curPredicate` (nor the parser state in general) is a stack, they are global state. Triple constructors save and restore their values, so the "stack" is implemented via the activation records of these constructors, such as the Triple Term constructor.